src/content/release-notes.mdx

```diff
@@ -5,6 +5,37 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.37.0
+
+3 October 2025
+
+This version is compatible with Kubernetes versions 1.31 to 1.33 \
+Okteto Chart release 1.36 is designed to work with [Okteto CLI 3.12.x](https://github.com/okteto/okteto/releases/tag/3.12.0)
+
+### Breaking Changes {#breaking-changes-1.37}
+
+- Removed the dependency on the Bitnami Redis Helm chart. Okteto now ships its own Kubernetes templates to deploy Redis <!-- 9663 -->
+
+### New Features {#new-features-1.37}
+
+- Added a centralized [Known Hosts feature to the Admin UI](admin/ssh-known-hosts.mdx) (Admin → Settings → Known Hosts). Admins can now pin trusted SSH host keys and disable automatic ssh-keyscan, ensuring secure, consistent cloning of repositories and submodules without custom runner images <!-- 9662 -->
+
+### Improvements {#improvements-1.37}
+
+- Improved error handling in the Okteto AI Agent UI for API errors and "Prompt too long" warnings <!-- 9636 -->
+- Temporal rate limit (QPS exceeded) errors now return as "progressing" instead of failing immediately <!-- 9664 -->
+- We've renamed AI Agent Fleets to Okteto AI throughout the product <!-- 9677 -->
+- Changed the [default value of `pullAlways` to **false**](self-hosted/helm-configuration.mdx#pullalways) for pods deployed in Okteto-managed namespaces. This speeds up pod creation when images are already present on the node, while still allowing `pullAlways` to be explicitly enabled if needed <!-- 9690 -->
```

Comment:
this is a breaking change, since it will change a behavior of Okteto for every self-hosted customer. could we move it up to the breaking changes section?